### PR TITLE
Enforce linting config

### PR DESCRIPTION
**Pull request by amilosmanli**
Enforces existing `tslint.json` linting config on the code.

Changes are made automatically by running `yarn lint --fix`

### CODE REVIEW

File: src/providers/lazy-loading/paramsToQuery.ts [hunk 1]
```diff
@@ -11,7 +11,7 @@ interface ParamsToQueryOptions {
 const defaultParamsToQueryOptions = {
   filters: true,
   sort: true,
-  pagination: true,
+  pagination: true
```
Comment (benwinding) [inline src/providers/lazy-loading/paramsToQuery.ts:14]: I personally prefer "trailing commas" (makes merging and rearranging easier) 

Are you able to put that into the `tslint.json` too? Not sure which rule changes it though 🤔 
```
"trailing-comma": ...
"comma-dangle": ...
```

**Review by benwinding — CHANGES_REQUESTED** [review]
Looking good, I just prefer "trailing/dangling commas"

**Comment by amilosmanli** [comment]
@benwinding we can merge this and then move to refactoring to modular firebase

**Review by benwinding — APPROVED** [review]
Perfect! Merging 👌

**Pull request merged**